website/docs/getting-started.md

```diff
@@ -4,191 +4,184 @@
+    ipVersion: 4
     allocationPools:
     - start: 192.168.1.5
       end: 192.168.1.60
     cidr: 192.168.1.0/24
-    ipVersion: IPv4
 EOF
 ```
 
 The controller will only attempt creating the subnet when the corresponding
-OpenStackNetwork exists:
+Network exists:
 
 ```plaintext
-$ kubectl get openstacksubnet
-NAME       READY   ERROR   STATUS
-subnet-1   False   False   Waiting for the following dependencies to be ready: network:default/network-1
+$ kubectl get subnets
+NAME       ID    AVAILABLE   MESSAGE                                       AGE
+subnet-1         False       Waiting for Network/network-1 to be created   4s
 ```
 
 Let's create it:
 
 ```yaml
 kubectl apply -f- <<EOF
 apiVersion: openstack.k-orc.cloud/v1alpha1
-kind: OpenStackNetwork
+kind: Network
 metadata:
   labels:
     app.kubernetes.io/name: openstacknetwork
     app.kubernetes.io/instance: openstacknetwork-gettingstarted
     app.kubernetes.io/part-of: gettingstarted
   name: network-1
 spec:
-  cloud: osp1
+  cloudCredentialsRef:
+    cloudName: openstack
+    secretName: openstack-clouds
+  managementPolicy: managed
+  networkRef: network-1
   resource:
-    name: network-1
+    description: |
+      Example network
+    tags:
+    - gettingstarted
 EOF
 ```
 
 After a few seconds, both resources become ready:
 
 ```plaintext
-$ kubectl get openstacknetwork
-NAME        READY   ERROR   STATUS
-network-1   True    False   Ready
+$ kubectl get networks
+NAME        ID                                     AVAILABLE   MESSAGE                           AGE
+network-1   5df739fb-2cdf-4d49-ad67-95fd36d99056   True        OpenStack resource is available   96s
 
-$ kubectl get openstacksubnet
-NAME       READY   ERROR   STATUS
-subnet-1   True    False   Ready
+$ kubectl get subnets
+NAME       ID                                     AVAILABLE   MESSAGE                           AGE
+subnet-1   bb1f0b74-0e79-4f77-b518-6a05a61662f0   True        OpenStack resource is available   2m42s
 ```
 
 The subnet can be inspected through its Kubernetes representation, under
 `.status.resource`:
 
 ```sh
-kubectl get openstacksubnet subnet-1 -o yaml
+kubectl get subnet subnet-1 -o yaml
 ```
 
 ```yaml
 apiVersion: openstack.k-orc.cloud/v1alpha1
-kind: OpenStackSubnet
+kind: Subnet
 metadata:
-  # [...]
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"openstack.k-orc.cloud/v1alpha1","kind":"Subnet","metadata":{"annotations":{},"labels":{"app.kubernetes.io/instance":"openstacksubnet-gettingstarted","app.kubernetes.io/name":"openstacksubnet","app.kubernetes.io/part-of":"gettingstarted"},"name":"subnet-1","namespace":"default"},"spec":{"cloudCredentialsRef":{"cloudName":"openstack","secretName":"openstack-clouds"},"managementPolicy":"managed","networkRef":"network-1","resource":{"allocationPools":[{"end":"192.168.1.60","start":"192.168.1.5"}],"cidr":"192.168.1.0/24","description":"Example subnet\n","ipVersion":4,"tags":["gettingstarted"]}}}
```

Comment:
Incidentally, avoiding this annotation is the primary reason I always use `--server-side` when creating resources!